src/ray/core_worker/store_provider/plasma_store_provider.cc

```diff
@@ -27,7 +27,6 @@ Status CoreWorkerPlasmaStoreProvider::SetClientOptions(std::string name,
 
 Status CoreWorkerPlasmaStoreProvider::Put(const RayObject &object,
                                           const ObjectID &object_id) {
-  RAY_CHECK(!object_id.IsDirectCallType());
```

Comment:
Why remove this?

Reply:
Oh I moved it into `Create`, which this one calls anyway.